scripts/requirements.txt

```diff
@@ -0,0 +1,2 @@
+cryptography
+intelhex
```

Comment:
Nitpick: this should go into the next patch ideally. It's fine by me to merge as-is though.

Reply:
Done

Reply:
I'm still seeing it in this same patch... ?

Reply:
Sorry, "done locally". Pushed now with all your comments addressed.

Reply:
Why would it go in the next patch? This is the patch that needs intelhex in order to run?

Reply:
@d3zd3z you're looking at the corrected revision of the series. It was previously in the patch that introduced requirements.txt.

Reply:
Ahh, thanks.